Podcast/TagButton.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TagButton.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 2/19/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class TagButton: UIButton {
+    
+    static func tagButton() -> TagButton {
```

Comment:
Why is this a static function? Why not just put this in the initializer for TabButton?